pennylane/circuit_graph.py

```diff
@@ -155,10 +155,10 @@ def __init__(self, ops, variable_deps):
         self.num_wires = 0
         """int: number of wires the circuit contains"""
         for k, op in enumerate(ops):
-            self.num_wires = max(self.num_wires, max(list(_flatten(op.wires))) + 1)
+            self.num_wires = max(self.num_wires, max(list(_flatten(op.wires.tolist()))) + 1)
             op.queue_idx = k  # store the queue index in the Operator
             for w in set(
-                _flatten(op.wires)
+                _flatten(op.wires.tolist())
```

Comment:
Again the case as above. In principle yes, but after #656 we can remove the ``_flatten`` function. I should have done that in that PR, thinking about it!

Reply:
Even then the ``tolist()`` is needed, because the iteration will pick ``w`` which are ``Wires`` of length one (remember, the ``__it__`` function returns Wire objects). And then the ``_grid`` in the next line will be made up of Wires objects and all hell breaks loose.